src/variables.js

```diff
@@ -92,7 +92,13 @@ module.exports = {
 			{ label: 'Remaining Minutes', name: 'remaining_mm' },
 			{ label: 'Remaining Seconds', name: 'remaining_ss' },
 			{ label: 'Remaining Fraction', name: 'remaining_ff' },
-		]
+		].concat(
+			Array(this.config.maxClips)
+				.fill( { label: undefined, name: undefined } )
+				.map( (e, i) => {
+					return { label: 'Clip ' + (i + 1), name: 'clip_' + (i + 1) }
+				})
+		)
```

Comment:
Variables should be dynamically created/updated based on the clips reported by PlayoutBee.

Reply:
I will adapt.